stories/chart.stories.tsx

```diff
@@ -222,7 +245,13 @@ storiesOf('HURUmap UI|Charts/LineChart', module)
               }
             }
           },
-          scatter: [{ size: 5, symbol: 'circle' }, { size: 5, symbol: 'plus' }]
+          group: {
+            // Line chart combines line and scatter hence best to define
+            // labels at group level
+            labels: datum => `${datum.x}\n${datum.geo} ${datum.y}`
```

Comment:
Does this mean we can't have normal labels on the points or above a bar in a bar chart?

Reply:
I don't see labels at the top in the designs (both Dominion and new HURUmap Designs),

----

I haven't tested it but may be defining `labelComponent` on the bar chart itself may allow extra labels.